core/unit_test/tools/TestTuning.cpp

```diff
@@ -53,8 +53,6 @@
 
 static size_t expectedNumberOfContextVariables;
 static int64_t expectedContextVariableValue;
-static std::unordered_map<size_t, Kokkos::Tools::Experimental::SetOrRange>
-    candidate_value_map;
```

Comment:
How comes we don't catch this in any of the CI builds?

Reply:
We only enable Tuning in the `OpenMPTarget` build where we don't use `-Werror`.

Reply:
OK and CUDA-10.1-Clang-Tidy

Reply:
Why doesn't the `CUDA-10.1-Clang-Tidy` build catch it?